solutions/advanced/cses-1695.mdx

```diff
@@ -89,95 +89,100 @@ int main() {
 ```java
 import java.util.*;
 
-public class PoliceChase {
-	static final int N = 505;
-	static int n, m;
-	static int[] lev = new int[N];
-	static int[][] c = new int[N][N];
-	static int[] seen = new int[N];
-	static Queue<Integer> q = new LinkedList<>();
-
-	// BFS to find level of each node
-	static boolean bfs() {
-		Arrays.fill(lev, 1, n + 1, -1);
-		lev[1] = 0;
-		q.add(1);
-		while (!q.isEmpty()) {
-			int u = q.poll();
-			for (int v = 1; v <= n; ++v) {
-				if (lev[v] == -1 && c[u][v] > 0) {
-					lev[v] = lev[u] + 1;
-					q.add(v);
-				}
-			}
-		}
-		return lev[n] != -1;
-	}
-
-	// DFS to find augmenting paths
-	static boolean dfs(int u) {
-		if (u == n) return true;
-		for (; seen[u] <= n; ++seen[u]) {
-			int v = seen[u];
-			if (lev[v] == lev[u] + 1 && c[u][v] > 0 && dfs(v)) {
-				c[u][v]--;
-				c[v][u]++;
-				return true;
-			}
-		}
-		return false;
-	}
-
-	// DFS to mark reachable nodes
-	static void dfs2(int u) {
-		lev[u] = 1;
-		for (int v = 1; v <= n; ++v) {
-			if (lev[v] == 0 && c[u][v] > 0) { dfs2(v); }
-		}
-	}
-
-	public static void main(String[] args) {
-		Scanner sc = new Scanner(System.in);
-		n = sc.nextInt();
-		m = sc.nextInt();
-
-		for (int i = 0; i < m; ++i) {
-			int u = sc.nextInt();
-			int v = sc.nextInt();
-			c[u][v]++;
-			c[v][u]++;
-		}
-
-		while (bfs()) {
-			Arrays.fill(seen, 1, n + 1, 1);
-			while (dfs(1));
-		}
-
-		Arrays.fill(lev, 1, n + 1, 0);
-		dfs2(1);
+public class PoliceChase { static final int N = 505; static int n, m; static
+int[] lev = new int[N]; static int[][] c = new int[N][N]; static int[] seen =
+new int[N]; static Queue<Integer> q = new LinkedList<>();
```

Comment:
absolutely wild formatting

Reply:
Apologies for the oversight, Corrected it now